### ISSUE TITLE
Would it be possible to implement for the debugger's console something similar to what was implemented for the terminal's accessibility buffer?

### ISSUE THREAD

**Issue by jvesouza** [microsoft/vscode#175879]
Type: <b>Feature Request</b>
cc: @meganrogge @isidorn 

When running or debugging an application through VSCode, it is possible to access the messages displayed by the application in the debugger console. I can navigate between messages using up and down arrows but I can't navigate a message using right and left arrows, and I can't select part of the message with the keyboard.

In my opinion, it would be interesting if something similar to what was implemented for the terminal's accessibility buffer were implemented for the debugger's console.

VS Code version: Code - Insiders 1.76.0-insider (92da9481c0904c6adfe372c12da3b7748d74bdcb, 2023-02-28T11:50:46.070Z)
OS version: Linux x64 6.1.14-200.fc37.x86_64
Modes:
Sandboxed: Yes

**Comment by isidorn** [comment]
This is a fair request, but I think the problem is that the debug console uses a Tree as an implementation detail. So each element is a tree element. And left/right navigation thus does not work as you would have expected.

Maybe a special mode in the debug console could help here.
Thus assigning to @roblourens as a feature request.

**Comment by VSCodeTriageBot** [comment]
This feature request is now a candidate for our backlog. The community has 60 days to [upvote](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
This feature request has not yet received the 20 community [upvotes](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) it takes to make to our backlog. 10 days to go. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
:slightly_frowning_face: In the last 60 days, this feature request has received less than 20 community upvotes and we closed it. Still a big Thank You to you for taking the time to create this issue! To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!
